lib/pdk/cli/module/generate.rb

```diff
@@ -0,0 +1,42 @@
+require 'tty-prompt'
+
+module PDK::CLI
+  @module_generate_cmd = @module_cmd.define_command do
+    name 'generate'
+    usage _('generate [options] <module_name>')
+    summary _('This command is now \'pdk new module\'.')
+    be_hidden
+
+    PDK::CLI.template_url_option(self)
+    PDK::CLI.skip_interview(self)
+
+    flag nil, 'skip-interview', _('When specified, skips interactive querying of metadata.')
```

Comment:
should make use of `PDK::CLI.skip_interview(self)`

Reply:
lol. it does. Looks like I did an errant Ctrl-R in vim to readd this line. It's adding skip-interview twice. :(